06_gpu_and_ml/yolo/yolo.py

```diff
@@ -0,0 +1,288 @@
+# ---
+# runtimes: ["runc", "gvisor"]
```

Comment:
fyi, we no longer use the `runtimes` flag. all workloads run on `gvisor` by default